src/lib/custom-icons/index.ts

```diff
@@ -26,6 +26,19 @@ const audio: IconDefinition = {
   ],
 }
 
+const caretDown: IconDefinition = {
+  prefix: 'fa-solid' as IconPrefix,
+  iconName: 'fa-caret-down' as IconName,
+  icon: [
+    320,
+    512,
+    [],
+    '',
+    // eslint-disable-next-line max-len
+    'M137.4 374.6c12.5 12.5 32.8 12.5 45.3 0l128-128c9.2-9.2 11.9-22.9 6.9-34.9s-16.6-19.8-29.6-19.8L32 192c-12.9 0-24.6 7.8-29.6 19.8s-2.2 25.7 6.9 34.9l128 128z',
+  ],
+}
+
```

Comment:
Not sure if this is the way we add icons.
But `<font-awesome-icon icon="fa-solid fa-caret-down" />` was giving an error.
Should we try updating the outdated fortawesome dependencies?